components/mixins/opportunity/opportunity-person-relation.schema.json

```diff
@@ -0,0 +1,33 @@
+{
+  "meta:license": [
+    "Copyright 2020 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/mixins/opportunity-person-relation",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "opportunity Person Relation Details",
+  "type": "object",
+  "meta:extensible": true,
+  "meta:abstract": true,
+  "meta:intendedToExtend": ["https://ns.adobe.com/xdm/context/opportunity-person"],
+  "description": "XDM business opportunity person relation details.",
+  "definitions": {
+    "opportunity-person-details": {
+      "properties": {}
+    }
+  },
+  "allOf": [
+    {
+      "$ref": "https://ns.adobe.com/xdm/common/extensible#/definitions/@context"
+    },
+    {
+      "$ref": "#/definitions/opportunity-person-relation"
+    },
+    {
+      "$ref": "https://ns.adobe.com/xdm/datatypes/b2b-opportunity-person"
```

Comment:
Same comment here. Why a b2b-opportunity-person datatype and not just fields defined in this mixin?

Also, should these opportunity->person mixins really be a generic global schema instead? Are these ever intended to be extended by the user?